src/main/java/fr/mieuxvoter/mj/DeliberatorInterface.java

```diff
@@ -3,16 +3,18 @@
 
 /**
  * A Deliberator takes in a poll's Tally,
```

Comment:
The code is properly commented. Do you think it makes sense to generate a doc? (like a ReadTheDoc)

Reply:
It always makes sense to generate a doc.

I'll add a new Issue for this.